docs/conf.py

```diff
@@ -33,15 +34,15 @@ def get_version(*file_paths):
     version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M)
     if version_match:
         return version_match.group(1)
-    raise RuntimeError('Unable to find version string.')
+    raise RuntimeError("Unable to find version string.")
```

Comment:
@xitij2000 There is quite a bit of formatting noise on this PR. Would it be possible to split the linting and feature into independent PRs or at least commits?